example/particle_tracers/particle_tracers.cpp

```diff
@@ -150,7 +150,7 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
 
   // Add swarm of tracer particles
   std::string swarm_name = "tracers";
-  Metadata swarm_metadata({Metadata::Provides});
+  Metadata swarm_metadata({Metadata::None});
```

Comment:
Right now we don't use any specific metadata for swarms, but `Provides` is required. This is normally the default, but `Metadata({})` is ambiguous and a compiler error so we need to pass *something*. `None` is correct since this doesn't have cell-specific data, but `Provides` is also correct... Honestly I'm not sure what the best solution to this right now is. We may ultimately want `Metadata` in `Swarm`s at least for the `Provides`/`Requires` flags with multiple packages.

Reply:
Hmm... what about being just extremely explicit. I.e., `swarm_metadata({Metadata::Provides, Metadata::None});`? That way it's clear it's both things?

Reply:
Switched to `swarm_metadata({Metadata::Provides, Metadata::None});` for all three particles examples